PocketKit/Sources/PocketKit/Collections/CollectionStory+Extensions.swift

```diff
@@ -0,0 +1,11 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Sync
+
+extension CollectionStory {
+    var isSaved: Bool {
+        item?.savedItem != nil && item?.savedItem?.isArchived == false
```

Comment:
nit nit: Alternative
```suggestion
        item?.savedItem?.isArchived == false
```

Reply:
Though I do have some concerns around asking if this item is saved and a nil counts as not saved.
Seems misleading that a nil item and an archived item would return the same result.

Reply:
Yeah, what you suggest works totally fine, I'd just leave it as it is for clarity.